config/jest.config.js

```diff
@@ -40,4 +38,11 @@ module.exports = {
     "ts",
     "tsx",
   ],
+  snapshotSerializers: ["enzyme-to-json/serializer"],
```

Comment:
Does this work with non-enzyme snapshots? Or are you only using enzyme for tests?

Reply:
It seems to work fine, with non `enzyme` snapshots, we are already doing it!